components/camel-kudu/src/main/resources/org/apache/camel/catalog/components/kudu.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
This is not useful for camel and it can be removed. Please have a look at the other component structure.